Makefile

```diff
@@ -122,8 +122,11 @@ e2e: #EXHELP Run the e2e tests.
 
 E2E_REGISTRY_NAME := docker-registry
 E2E_REGISTRY_NAMESPACE := operator-controller-e2e
+DNS_NAME           := $(E2E_REGISTRY_NAME).$(E2E_REGISTRY_NAMESPACE).svc.cluster.local
+
 export REG_PKG_NAME := registry-operator
 export CATALOG_IMG := $(E2E_REGISTRY_NAME).$(E2E_REGISTRY_NAMESPACE).svc:5000/test-catalog:e2e
+export REGISTRY_ROOT := $(E2E_REGISTRY_NAME).$(E2E_REGISTRY_NAMESPACE).svc:5000
```

Comment:
Is there a reason why we need both `DNS_NAME` and `REGISTRY_ROOT`?

Reply:
`CATALOG_IMG` is used in both the `Makefile` and the `setup.sh` script for the dev-ext-e2e, where as `REGISTRY_ROOT` is only needed by `setup.sh`. I suppose one can be defined by the other.

Reply:
..._and_ it's used inside the ext-dev-e2e golang files...

Reply:
Are you talking about `CATALOG_IMG`?

I'm asking about `DNS_NAME` and `REGISTRY_ROOT` which seem to be virtually the same thing and both are just introduced in this PR.

Differences:
* `DNS_NAME` for some reason uses `cluster.local` DNS when `REGISTRY_ROOT` uses `svc`
* `DNS_NAME` does not include port, but all the usages of `DNS_NAME` (see below in `Makefile`) then add the port.

Reply:
Ah... yeah, that can be cleaned up.

Reply:
I think this is now resolved.